opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java

```diff
@@ -215,10 +217,14 @@ private HistoryRevResult getHistoryRev(
              * not exist or internal git error occured.
              */
             result.success = (status == 0);
-        } catch (Exception exp) {
+        } catch (Exception exception) {
```

Comment:
you mentioned that the out-of-bounds exception was swalloed by this catch. Is using `Exception` maybe too broad then ?

Reply:
or move the `filename` assignment outside of the `try` block ?

Reply:
No to the problem was it wasn't even logged with the bad logger parameters.

Reply:
I just changed it so it's being logged

Reply:
Okay. Still large try/catch blocks with generic catch make me nervous.